lib/opensusebasetest.pm

```diff
@@ -858,6 +859,16 @@ sub grub_select {
     elsif (!get_var('S390_ZKVM')) {
         # confirm default choice
         send_key 'ret';
+        if (get_var('USE_SUPPORT_SERVER') && is_aarch64 && is_opensuse)
```

Comment:
perhaps with the first condition is enough in case more slower archs would behave the same, wdyt?

Reply:
We run this test on SLES and 64bit TW, where this workaround is not needed, as I also call `wait_still_screen` below which will waste some seconds, so I would keep it limited to this scenario only.